tac/governing-documents/release-signing.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title:  Release Signing
+parent: Governing Documents
+grand_parent: LF Decentralized Trust TAC
+nav_order: 1
```

Comment:
```suggestion
nav_order: 10
```
Not sure the right number, but I am guessing 10 is close.

Reply:
Do you think there's a possibility for us to improve the structure with sub-folders? For instance, project reports may have annual report and quarterly report in them. Security may have governing documents related to software supplychain, the other process related to having a representative from the project.

Reply:
+1 for trying to avoid too many/too deep folder structures in general